cfy_manager/components/validations.py

```diff
@@ -90,6 +91,45 @@ def _validate_supported_distros():
         )
 
 
+def _validate_private_ip():
+    logger.info('Validating private IP address...')
+    private_ip = config[MANAGER][PRIVATE_IP]
+    all_addresses = set()
+    found = False
+    all_interfaces = netifaces.interfaces()
+    for interface in all_interfaces:
+        int_addresses = netifaces.ifaddresses(interface)
+        if not int_addresses:
+            logger.debug('Could not find any addresses for interface {}'.
+                format(interface))
+            continue
+        inet_addresses = int_addresses.get(netifaces.AF_INET)
+        if not inet_addresses:
+            logger.debug('No AF_INET addresses found for interface {}'.format(
+                interface))
+            continue
```

Comment:
Does `int_addresses.get(netifaces.AF_INET)` return `None` or an empty list? If it's an empty list, then the `continue` is not necessary.

Reply:
`int_addresses` is a `dict`, so `None` will be returned. Keeping it as is.